core/services/job/orm.go

```diff
@@ -132,13 +139,27 @@ func (o *orm) CreateJob(ctx context.Context, jobSpec *models.JobSpecV2, taskDAG
 	defer cancel()
 
 	return postgres.GormTransaction(ctx, o.db, func(tx *gorm.DB) error {
+		// TODO(connor): this needs to accept the tx to be transactional?
```

Comment:
Yes we really ought to fix this before merging

Reply:
done